### PR TITLE
Updates for bazel and the cloud-create script

### PR DESCRIPTION
**Pull request by chrislovecnm**
- Parameterizing Builds
 - More tweaks to cloud-create
- Fixed the Makefile
- Started to remove dev cluster
- Work on getting CI to run

Dev cluster is not functional at this point since elasticsearch is not loading  correctly.  Staging is functioning.

### CODE REVIEW

**Comment by chrislovecnm** [comment]
please test this

**Comment by chrislovecnm** [comment]
test this please

**Comment by chrislovecnm** [comment]
test this please

File: scripts/wait-on-pyrios.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# In order to not expose this project to the internet with a Cloud Load Balancer
+# we are instead port-forwarding to the node and exposing the pod port
+
+# "---------------------------------------------------------"
+# "-                                                       -"
+# "-  port-forward to pyrios pod of the cloud GKE cluster  -"
```
Comment (chrislovecnm) [inline scripts/wait-on-pyrios.sh]: Copy pasta error ...
```suggestion
# "-  test that pyrios pod of the cloud GKE cluster has started  -"
```

File: Makefile [hunk 2]
```diff
@@ -95,11 +118,11 @@ configure:
 # todo: migrate this to bazel. (pyrios and ui)
```
Comment (geojaz) [inline Makefile]: this can probably go now since *-create uses bazel

File: scripts/cloud-create.sh [hunk 3]
```diff
@@ -26,34 +26,46 @@ set -o nounset
...
+	--from-literal=ES_SERVER="$LB_IP" || true
+
+if [[ "$(command -v bazel >/dev/null 2>&1 )" ]] ; then
+    echo >&2 "pyrios is currently built and managed via bazel which is not installed."
+    echo >&2 "in the future, we will try to provide fall back options using kubectl (boring!)"
+    echo "we are not deploying pyrios right now. get your bazel on first!"
+    exit 1
+else
+    echo "building and deploying pyrios and pyrios-ui"
+    # TODO need to fix the context for the repo
```
Comment (geojaz) [inline scripts/cloud-create.sh]: this can be removed

**Pull request merged**